### ISSUE TITLE
Subscriptable types split incorrectly

### ISSUE THREAD

**Issue by als-wobe** [kozhushman/pydocstring#5]
https://github.com/kozhushman/pydocstring/blob/3fbc12ae2479b5e46e07d897e4364019413e37a7/src/extension.ts#L124

Subscriptable types like `param: Dict[str, str]` are split into `param (Dict[str):` and `str] (undefined):`

Reproduction:

``````python
from typing import Dict

def test(p: Dict[str, str]):
    ...
``````

**Comment by kozhushman** [comment]
Thanks! Can you also post block of python code, for easy reproducing?

**Comment by als-wobe** [comment]
I added a minimal reproduction

**Comment by kozhushman** [comment]
Fixed in release 0.1.0
